dev/bots/test/analyze-snippet-code-test-dart-ui/ui.dart

```diff
@@ -5,8 +5,7 @@
 // @dart = 2.12
 
 // This is a dummy dart:ui package for the sample code analyzer tests to use.
```

Comment:
Could just turn this line into a doc comment!

Reply:
Oh, yeah! I'll do that!